src/components/React/Controls.js

```diff
@@ -50,180 +33,80 @@ export default connect(mapStateToProps, {
-              />
-              <br />
-            </div>
-
-            <label>
-              <br />
-              Enable Color Map &nbsp;
-              <Checkbox
-                id="colorMapCheckBox"
-                tooltip="Enabled when a Volume is loaded"
-                onChange={this.handleCheckBoxInputChange}
-                checked={this.state.activateColorMapping}
-              ></Checkbox>
-            </label>
-
-            <div
-              style={
-                this.state.activateColorMapping && this.props.volumeData !== ""
-                  ? {}
-                  : { display: "none" }
-              }
-            >
-              <ColorMapControl width="255" />
-              <OpacityControl width="250" />
-            </div>
-          </div>
-
-          <div>
-            <label>
-              X Slide <br />
-            </label>
-            <Range
-              disabled={
-                this.props.volumeData === "" ||
-                this.props.volumeData === undefined
-                  ? true
-                  : false
-              }
-              allowCross={false}
-              step={0.0009}
-              defaultValue={[0, 1]}
-              min={0}
-              max={1}
-              onChange={this.xSlideHandleChange}
-            />
-            <br />
-
-            <label>
-              {" "}
-              Y Slide <br />
-            </label>
-            <Range
-              disabled={
-                this.props.volumeData === "" ||
-                this.props.volumeData === undefined
-                  ? true
-                  : false
-              }
-              allowCross={false}
-              step={0.0009}
-              defaultValue={[0, 1]}
-              min={0}
-              max={1}
-              onChange={this.ySlideHandleChange}
-            />
-            <br />
-
-            <label>
-              {" "}
-              Z Slide <br />{" "}
-            </label>
-            <Range
-              disabled={
-                this.props.volumeData === "" ||
-                this.props.volumeData === undefined
-                  ? true
-                  : false
-              }
-              allowCross={false}
-              step={0.0009}
-              defaultValue={[0, 1]}
-              min={0}
-              max={1}
-              onChange={this.zSlideHandleChange}
-            />
-          </div>
-        </div>
+        <Container fluid id="controls">
+          <Row className="my-3">
+            <ColorMapControl width="250" />
+          </Row>
+          <Row className="my-3">
+            <OpacityControl width="250" />
+          </Row>
+
+          <Row className="mt-5">
+            <h4>Clip</h4>
+            <Form className="fullWidth">
+              <Form.Group>
+                <Form.Label> X </Form.Label>
+                <Range
+                  allowCross={false}
+                  step={0.0009}
+                  defaultValue={[0, 1]}
```

Comment:
should the `defaultValue` on all of these also be the `range`?

Reply:
So add `defaultValue` to `config.range`?

Reply:
I think it might be better to have it as `defaultValue={[range.min, range.max]}` in case those values ever change

Reply:
yup! that's what I meant - sorry for the confusion